test/test_crypto.py

```diff
@@ -75,6 +76,9 @@ def test_md5_iterated(self):
         expected = b'\x90\x01\x50\x98\x3c\xd2\x4f\xb0\xd6\x96\x3f\x7d\x28\xe1\x7f\x72'
         self.assertEqual(expected, digest)
 
+    def test_is_tls_cipher_supported(self):
```

Comment:
trivial: put this test in `test_io.py` since it's testing something from `io.py`

this file is for testing stuff from `crypto.py`